xdg-settings uses the ${parameter:offset} substring expansion,
which is not available in posix shell

--- scripts/xdg-settings.in	2018-05-10 17:02:31.000000000 +0200
+++ -	2021-06-12 21:17:34.555007469 +0200
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #---------------------------------------------
 #   xdg-settings
 #
